python/mxnet/metric.py

```diff
@@ -592,8 +592,7 @@ def update(self, labels, preds):
 
 
 class _BinaryClassificationMetrics(object):
-    """
-    Private container class for classification metric statistics. True/false positive and
+    """Private container class for classification metric statistics. True/false positive and
```

Comment:
Add two newlines prior to "True/false positive and" https://www.python.org/dev/peps/pep-0257/